components/esp_eth/src/esp_eth_mac_w5500.c

```diff
@@ -233,6 +233,8 @@ static esp_err_t w5500_verify_id(emac_w5500_t *emac)
     // W5500 doesn't have chip ID, we just print the version number instead
     ESP_LOGI(TAG, "version=%x", version);
```

Comment:
We should change the comment and could remove the LOGI.
```suggestion
    // W5500 doesn't have chip ID, we check the version number instead
```